### ISSUE TITLE
destroy_cmd_entrypoint still uses absolute path instead of relative module path

### ISSUE THREAD

**Issue by bongtavas** [terraform-google-modules/terraform-google-cloud-operations#48]
### TL;DR

The `destroy_cmd_entrypoint` when using `terraform` apply always changes since it's using the absolute path instead of relative module path.

![image](https://user-images.githubusercontent.com/4264560/192712949-5385fa47-d120-46e3-a752-3e486d23aeea.png)

### Expected behavior

`destroy_cmd_entrypoint` should use module relative path to locate the destroy script.

### Observed behavior

`destroy_cmd_entrypoint` is using absolute path making `terraform apply` detect there is a change even though it was only ran in a different machine.

### Terraform Configuration

```hcl
nmodule "onegie_agent_policy" {
  source     = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version    = "~> 0.2.3"

  project_id = var.gcp_project_id
  policy_id  = "ops-agents-policy"
  agent_rules = [
    {
      type               = "ops-agent"
      version            = "current-major"
      package_state      = "installed"
      enable_autoupgrade = true
    },
  ]
  instances = var.instances
  os_types = [
    {
      short_name = "debian"
      version = "10"
    }
  ]
}
```

### Terraform Version

```sh
Terraform v1.0.10
on darwin_arm64
```
